src/views/TwabRewards/hooks/useTwabRewardsSubgraphClient.ts

```diff
@@ -0,0 +1,25 @@
+import { GraphQLClient } from 'graphql-request'
+
+import { CHAIN_ID } from '@constants/misc'
+import { theGraphCustomFetch } from '@utils/theGraphCustomFetch'
+
+const POOLTOGETHER_GOVERNANCE_GRAPH_URIS = {
+  [CHAIN_ID.rinkeby]: '',
```

Comment:
Do we need the rest of these? Is `POOLTOGETHER_GOVERNANCE_GRAPH_URIS` the right name?

Reply:
Yes, and nope. Funny all the copy and paste things my brain simply misses.